android/MobileSdk/src/main/java/com/spruceid/mobile/sdk/nfc/NfcListenManager.kt

```diff
@@ -0,0 +1,81 @@
+package com.spruceid.mobile.sdk.nfc
+
+import android.content.ComponentName
+import android.content.Context
+import android.nfc.NfcAdapter
+import android.nfc.cardemulation.CardEmulation
+import android.util.Log
+
+val APDU_AID_MDOC = "A0000002480400"
+val APDU_AID_NDEF_APPLICATION = "D2760000850101"
```

Comment:
Could you add references/links to the definition of those values? It's probably easy to find but it can be helpful